core/src/main/java/com/xmartlabs/base/core/log/analytics/AnalyticTrackable.java

```diff
@@ -0,0 +1,15 @@
+package com.xmartlabs.base.core.log.analytics;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.Map;
+
+public interface AnalyticTrackable<T> {
```

Comment:
The thing is that it can be a screen or an event. I think event was more suitable here, but it lets you think it's always an event when screens are also "trackable"